### ISSUE TITLE
When Extending Theme It gets duplicated multiple times

### ISSUE THREAD

**Issue by misilot** [DSpace/dspace-angular#2346]
**Describe the bug**
When extending the theme to overwrite it duplicates all the elements multiple times. This is with DSpace 7.6. 

So what I did was copy the dspace theme, made it called "theme1" and made my normal changes there. I then created theme2 and want to apply it to specific collections to add additional metadata fields to some of the views. 

It looks like it's rendering the original version, and the extended theme? See the photo below

![image](https://github.com/DSpace/dspace-angular/assets/1446856/1daed516-d3e2-45c3-aa99-3840da918952)

**To Reproduce**
1. Create custom theme by copying the dspace theme (theme1)
2. Enable your new theme1
3. Create a new theme (theme2) that you want to change only `src/themes/theme2/app/item-page/simple/item-types/untyped-item/untyped-item.component.html` in
4. Enable themes in config.yml
```yml
themes:
  - name: theme2
    extends: theme1
    uuid: '6d684812-6d29-4564-8b51-f5670957936b'

  # default theme, based on DSpace
  - name: theme1
```

**Expected behavior**
I would it to only change the elements defined in

**Comment by misilot** [comment]
I made the repo public https://github.com/kstatelibrariesit/dspace-angular/tree/etdr-theme/src/themes/etdr, in case it might make it easier to find what I might have messed up, if it's a user error.

**Comment by tantz001** [comment]
We are experiencing the same bug, and I have some additional info.  If I select a themed community, for example, via it's handle (/handle/12345/3434), the themed page renders correctly even though the url reverts to its /communities/uuid form.  I can even refresh the page successfully.  This is regardless of whether I use the handle or the uuid as the theme selector.  If I select a page via its uuid, I experience the duplication of page elements as described above.

**Comment by alexandrevryghem** [comment]
I would like to claim this issue

**Comment by alexandrevryghem** [comment]
Atmire would like to claim this issue

**Comment by tdonohue** [comment]
@alexandrevryghem : Sounds great! Thanks, I'll assign this to you for your team to work on.

**Comment by tdonohue** [comment]
For those following this issue, it now has a possible fix in #2395  Testers are welcome/encouraged! (The fix is also rather small, so it looks like you could copy it into your 7.6 instance if you are impacted)

**Comment by misilot** [comment]
I forgot to circle back that #2395 does fix the duplication issue I was seeing! Thank you @alexandrevryghem!
